packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -0,0 +1,167 @@
+# Accordion
+
+An accordion is used to allow users to toggle between different content sections in a compact vertical stack.
+
+This document outlines the API of `Accordion` component.
+
+<div align="center">
+  <img src="./accordion-thumbnail.png" width="380" alt="" />
+</div>
+
+## Design
+
+- [Figma - Accordion](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-619941&t=Rcrabprdzrz7JdId-0)
+
+## Interactions
+
+### Accordion open
+
+https://github.com/razorpay/blade/assets/6682655/1a02d680-1832-4a70-a0a3-1a1f56f2b8b9
+
+### Accordion close
+
+https://github.com/razorpay/blade/assets/6682655/b54eaeb9-7257-4da4-bf3b-89e10785e4f9
+
+## API
+
+Annotated component with props:
+
+![Accordion breakdown](./accordion-breakdown.png)
+
+Sample usage, composes `Accordion` and `AccordionItem`:
+
+```jsx
+import { Accordion, AccordionItem } from '@razorpay/blade';
+
+<Accordion showNumberPrefix defaultExpandedIndex={0}>
+  <AccordionItem
+    title="Can I create linked accounts using Route?"
+    description="You can use Razorpay Route from the Dashboard or using APIs to transfer money to customers."
+  >
+    <div>Slot</div>
+  </AccordionItem>
+  <AccordionItem title="How can I transfer money to customers?" description="Just use Razorpay" />
+</Accordion>;
+```
+
+### Accordion
+
+| Prop                 | Type       | Default     | Description                                                    | Required |
+| -------------------- | ---------- | ----------- | -------------------------------------------------------------- | -------- |
+| defaultExpandedIndex | `number`   | `undefined` | Makes the passed item index expanded by default (uncontrolled) |          |
```

Comment:
There might be contextual cases where the most important item might not be the first one so proposed this as a `number` rather than a `boolean` that only allows default expanding the first item